Makefile

```diff
@@ -0,0 +1,12 @@
+ifndef FH_SYSTEM_DUMP_TOOL_VERSION
```

Comment:
This allows us to override the version during build, either via environment variable or extra argument to `make`.

```bash
# 1
make FH_SYSTEM_DUMP_TOOL_VERSION=v1.0.0

# 2
FH_SYSTEM_DUMP_TOOL_VERSION=v1.0.0 make

# 3
export FH_SYSTEM_DUMP_TOOL_VERSION=v1.0.0
make
```

@philbrookes @vibe13, I went with a rather long variable name to namespace it properly (instead of calling it `VERSION`) without introducing alternative names or acronyms to the tool (thus not calling it `FH_SDT_VERSION`, for instance). I'm open to naming suggestions if there is a pattern to be followed.